client/src/components/Home/style.css

```diff
@@ -0,0 +1,33 @@
+.home {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 70%;
+}
+
+.home--heading {
+  font-size: 1.02rem;
+  text-align: center;
+}
+
+.home--buttons-container {
+  width: 100%;
+  padding: 1rem;
+}
+
+.home--button {
+  margin-bottom: 1.3rem;
+  border-radius: 3px;
+  background-color: #D65050;
+  border: none;
+  color: #ffffff;
+  transition: background-color 2s ease;
+}
+
+.home--button:hover {
+  background-color: rgba(214, 80, 80, 0.9);
+  color: #ffffff;
```

Comment:
useless line
```suggestion
  color: #ffffff;
```

Reply:
I used the button from ant design which has a blue color on hover. That's why I give it white color on hover to override the blue color